.azure/scripts/check_docs.sh

```diff
@@ -25,7 +25,7 @@ function grep_check {
 # Check for latin abbrevs
 grep_check '[^[:alpha:]](e\.g\.|eg)[^[:alpha:]]' "Replace 'e.g'. with 'for example, '"
 grep_check '[^[:alpha:]](i\.e\.|ie)[^[:alpha:]]' "Replace 'i.e'. with 'that is, '"
-grep_check '[^[:alpha:]]etc\.[^[:alpha:]]?' "Replace 'etc.'. with ' and so on.'"
+grep_check '[^[:alpha:]]etc[\.\n\r\s]' "Replace 'etc.'. with ' and so on.'"
```

Comment:
Is this change still needed? I think the issue was fixed in #3988 in the meantime.

Reply:
Looks like with #3988 it's not needed. I'll rebase the PR ...